src/main/java/no/ndla/taxonomy/rest/v1/dtos/nodes/searchapi/SearchableTaxonomyResourceType.java

```diff
@@ -14,6 +14,9 @@ public class SearchableTaxonomyResourceType {
     @JsonProperty
     private HashMap<String, String> name;
 
+    public SearchableTaxonomyResourceType() {
+    }
```

Comment:
Hvor brukes denne?

Reply:
Trengs kun for at serialisering til json skal funke.